src/plugins/intel_cpu/tests/functional/test_utils/cpu_test_utils.hpp

```diff
@@ -177,6 +177,8 @@ const std::map<std::string, std::string> cpuBF16PluginConfig =
 
 // utility functions
 std::vector<CPUSpecificParams> filterCPUSpecificParams(const std::vector<CPUSpecificParams>& paramsVector);
+std::vector<CPUSpecificParams> filterCPUInfo(const std::vector<CPUSpecificParams>& CPUParams);
```

Comment:
I think we should have a separate file, called filter_cpu_info.cpp and a separate header filter_cpu_info.hpp

Reply:
implemented